tests/Unit/Domain/CoordinateMaps/Test_Wedge2D.cpp

```diff
@@ -162,18 +166,12 @@ void test_equality() {
       CoordinateMaps::Wedge2D(0.2, 4.0, 0.3, 1.0, OrientationMap<2>{}, true);
   const auto wedge2d_outer_circularity_changed =
       CoordinateMaps::Wedge2D(0.2, 4.0, 0.0, 0.9, OrientationMap<2>{}, true);
-  const auto wedge2d_orientation_map_changed = CoordinateMaps::Wedge2D(
-      0.2, 4.0, 0.0, 1.0,
-      OrientationMap<2>{std::array<Direction<2>, 2>{
-          {Direction<2>::upper_eta(), Direction<2>::upper_xi()}}},
-      true);
```

Comment:
For my understanding: why are you removing this vs keeping the check but changing it so it uses a valid orientation map?